### DISCUSSION TITLE
how to send a email

### DISCUSSION THREAD

**Discussion by zhedahe** [mjs/imapclient#528] [Q&A] [answered]
hi, everybody:
     I read whole repo code, and search for how to send a email by imapclient in internet, but less info about the result, is there a sample for reference? thanks a lot.

**Comment by JohnVillalovos** [comment, marked as answer]
It receives email. It doesn't send email. Check the smtplib library which is part of Python

**Reply by zhedahe** [reply]
ok, thanks for help

**Comment by mjs** [comment]
The IMAP protocol is for receiving email. You want SMTP (as per @JohnVillalovos 's smtplib suggestion).

**Reply by zhedahe** [reply]
got it, thank you.
